src/components/DiffView/DiffView.test.tsx

```diff
@@ -0,0 +1,46 @@
+/// <reference path="index.d.ts"/>
+
+import { shallow } from 'enzyme';
+import React from 'react';
+import { parseDiff } from 'react-diff-view';
+
+import DiffView from '.';
+
+describe(__filename, () => {
+  const basicDiff = `diff --git a/src/components/DiffView.test.tsx b/src/components/DiffView.test.tsx
```

Comment:
How about using fixture files for the diffs? This one is ~10 lines long, but we might have larger diffs in the future. We could maybe version diff files instead?

Reply:
Yes, that's a good idea. The reason I just inlined it here is that I actually expect that the need for this will go away. I imagine that we'll have helpers that give us response data from the APIs, much as we have for addons-frontend, and at least one of those would contain diff information (I believe), so when we have that we won't need to store a sample diff elsewhere.

So because this is temporary, I felt like it made sense to just inline it.